server.go

```diff
@@ -51,14 +52,8 @@ func (s *Server) ListenAndServe() error {
 
 // Options represent server options
 type Options struct {
-	// MaxConcurrentStreams will apply a limit on the number of concurrent streams to each ServerTransport.
 	MaxConcurrentStreams uint32
-	// MaxMessageSize sets the max message size in bytes for inbound mesages.
-	// If this is not set, gRPC uses the default 4MB.
-	MaxMessageSize int
-
-	SkipCompression     bool
```

Comment:
soo, Compression on server level is now deprecated, it is something that needs to be enabled on a call level, i.e. when making an API call from the client https://godoc.org/google.golang.org/grpc#UseCompressor should be passed when we think it's worth the effort